tests/test_strtobool.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from decouple import strtobool
+
+
+def test_true_values():
+    true_list = ["y", "yes", "t", "true", "on", "1"]
+    for item in true_list:
+        assert strtobool(item) == 1
+
+
+def test_false_values():
+    false_list = ["n", "no", "f", "false", "off", "0"]
+    for item in false_list:
+        assert strtobool(item) == 0
+
+
+@pytest.mark.parametrize(
+    "test_input,expected",
+    [
+        ("Invalid_Value_1", "invalid truth value Invalid_Value_1"),
```

Comment:
It is testing the same scenario 3 times, I think only one invalid test would be ok.